biz.aQute.bndlib.tests/src/test/component/DSAnnotationTest.java

```diff
@@ -2860,6 +2860,49 @@ public void testPrefixCollisionExtraAttributesDefaultPrefix() throws Exception {
 
 	}
 
+	@XMLAttribute(namespace = "override", prefix = "", embedIn = "http://www.osgi.org/xmlns/scr/*")
+	@Retention(RetentionPolicy.CLASS)
+	@Target({
+		ElementType.TYPE
+	})
+	@interface TestSpecReplace {
+		boolean immediate() default false;
+
+		boolean enabled() default true;
+	}
+
+	@TestSpecReplace
+	@Component(immediate = true, enabled = false)
```

Comment:
I can see wanting to maybe override the defaults but it seems wrong to override specified values. This seems very surprising. I specified immediate=true but get false because of some other annotation specified elsewhere. I don't like that.